test/Sema/availability_renamed.swift

```diff
@@ -0,0 +1,87 @@
+// RUN: %target-parse-verify-swift
+
+//-- unnamed closure => named closure
+
+@available(*, unavailable, renamed: "newFn1(arg:fn:)")
+func oldFunc1(_ x: Int, _ closure: () -> Int) {}
+// expected-note@-1 2{{'oldFunc1' has been explicitly marked unavailable here}}
+
+oldFunc1(0) { 0 }
+// expected-error@-1 {{'oldFunc1' has been renamed to 'newFn1(arg:fn:)'}} {{1-9=newFn1}} {{10-10=arg: }} {{none}}
+oldFunc1(0, { 0 })
+// expected-error@-1 {{'oldFunc1' has been renamed to 'newFn1(arg:fn:)'}} {{1-9=newFn1}} {{10-10=arg: }} {{13-13=fn: }} {{none}}
+
+@available(*, unavailable, renamed: "newFn1(arg:fn:)")
+func oldFunc1_2(x: Int, _ closure: () -> Int) {}
+// expected-note@-1 2{{'oldFunc1_2(x:_:)' has been explicitly marked unavailable here}}
+
+oldFunc1_2(x: 0) { 0 }
+// expected-error@-1 {{'oldFunc1_2(x:_:)' has been renamed to 'newFn1(arg:fn:)'}} {{1-11=newFn1}} {{12-13=arg}} {{none}}
+oldFunc1_2(x: 0, { 0 })
+// expected-error@-1 {{'oldFunc1_2(x:_:)' has been renamed to 'newFn1(arg:fn:)'}} {{1-11=newFn1}} {{12-13=arg}} {{18-18=fn: }} {{none}}
+
+@available(*, unavailable, renamed: "newFn1_3(fn:)")
+func oldFunc1_3(_ closure: () -> Int) {}
+// expected-note@-1 3{{'oldFunc1_3' has been explicitly marked unavailable here}}
+
+oldFunc1_3 { 0 }
+// expected-error@-1 {{'oldFunc1_3' has been renamed to 'newFn1_3(fn:)'}} {{1-11=newFn1_3}} {{none}}
```

Comment:
> `{{none}}` should only be used when there are no fix-its; if there are any to check, it's taken as an exhaustive set.

I'm not sure. 😕
Without `{{none}}`, this test passes even without my change:  `{{1-11=newFn1_3}} {{12-12=fn: }}`